reana_server/config.py

```diff
@@ -68,6 +68,11 @@
 SUPPORTED_COMPUTE_BACKENDS = json.loads(os.getenv("REANA_COMPUTE_BACKENDS", "[]")) or []
 """List of supported compute backends."""
 
+DEFAULT_NAME_FOR_LAUNCHED_WORKFLOWS = "remote_workflow"
```

Comment:
You can simply use `workflow` as the default name when nothing is specified. Just as we do from CLI client. 

I'll create a separate issue for generating nice user-visible names for each external source.  (So that this PR would simply use it later.)